backend/src/impl/tasks.py

```diff
@@ -127,6 +127,11 @@ class TaskCategory:
                 name=TaskType.qa_open_domain,
                 description="Answer a question in an open domain",
             ),
+            Task(
```

Comment:
Is this change (and another one below) related to the purpose of this pull request?